manifests/install/charts/as-a-second-scheduler/templates/crd.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Can you move this to manifests/install/charts/as-a-second-scheduler/crds/podgroup.yaml?

Refs:
- https://helm.sh/docs/chart_best_practices/custom_resource_definitions/
- https://github.com/helm/community/blob/main/hips/hip-0011.md